support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/CassandraTicketRegistryDao.java

```diff
@@ -0,0 +1,29 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.ticket.Ticket;
+
+import java.util.stream.Stream;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.2.0
+ */
+public interface CassandraTicketRegistryDao {
```

Comment:
This interface likely needs to entirely go away as operations listed here are almost exact duplicates of the registry itself.